hyperbolic-rs/item_graph.py

```diff
@@ -74,7 +75,7 @@ def build_item_embeds(item_text, use_url, weight_first_embedding=False):
         embs = use_model(sents)
 
         weights = np.ones([1, len(embs)])
-        if weight_first_embedding:  # gives higher weight to title in keen case
+        if weight_first_embedding:  # gives higher weight to first embedding
```

Comment:
But then the logic of line: https://github.com/PAIR-code/recommendation-rudders/blob/feature/amazon/hyperbolic-rs/item_graph.py#L223
has to be changed to floats instead of booleans. 
I prefer to keep it as a boolean option from outside the function, and the mathematical logic inside the function.
I will change it to default to a value so the if is not inside the loop.
Thanks!